dask_kubernetes/operator/kubecluster/kubecluster.py

```diff
@@ -224,6 +228,7 @@ def __init__(
         self.idle_timeout = dask.config.get(
             "kubernetes.idle-timeout", override_with=idle_timeout
         )
+        self.quiet = quiet
```

Comment:
I think we should remove this and instead pass `quiet` along to the `super().__init__(..., quiet=quiet)` further down.

Reply:
Duh, heh. Not sure why I didn't think of that. Done.